framework/src/base/NonlinearSystem.C

```diff
@@ -870,6 +870,7 @@ NonlinearSystem::enforceNodalConstraintsJacobian(SparseMatrix<Number> & jacobian
       }
     }
   }
+  jacobian.close();
```

Comment:
Someone else probably knows better than me, but I think it's OK.  We're not (at least currently) looking up the value of the Jacobian in nc->computeJacobian(), just adding into it.  The problem was that later on, outside this method, we were looking up the value of an entry when the jacobian hadn't been closed.